AFAmazonS3Manager/AFAmazonS3RequestSerializer.m

```diff
@@ -32,7 +32,7 @@
 NSString * const AFAmazonS3EUCentral1Region = @"s3-eu-central-1.amazonaws.com";
 NSString * const AFAmazonS3APSoutheast1Region = @"s3-ap-southeast-1.amazonaws.com";
 NSString * const AFAmazonS3APSoutheast2Region = @"s3-ap-southeast-2.amazonaws.com";
-NSString * const AFAmazonS3APNortheast1Region = @"s3-ap-northeast-1.amazonaws.com";
+NSString * const AFAmazonS3APNortheast2Region = @"s3-ap-northeast-1.amazonaws.com";
```

Comment:
Why did you change the name on this constant?

Reply:
Check the header file, it's AFAmazonS3APNortheast2Region.

Reply:
ok, just curious, since the data center number doesn't match the constant. Since I have no affiliation with this, I don't really care.

Thanks for doing this, I will test this out if my client approves the upgrade to AFNetworking 3

Reply:
you're welcome!  please feel free to comment if has any bugs